fluentlenium-core/src/main/java/org/fluentlenium/core/Fluent.java

```diff
@@ -360,6 +360,32 @@ public Fluent goTo(String url) {
         return this;
     }
 
+    /**
+     * Open the url page in a new tab
+     *
+     * @param url
+     */
+    public Fluent goToInNewTab(String url) {
```

Comment:
Why are we using URL as string? Java has URL object to work w/ such things and below you are creating URI from string... I think it will be better for this method to accept URL object.